draft-ietf-sframe-enc.md

```diff
@@ -908,52 +894,52 @@ protect against such attacks.
 ### Simulcast
 
 When using simulcast, the same input image will produce N different encoded
-frames (one per simulcast layer) which would be processed independently by the
-frame encryptor and assigned an unique counter for each.
+frames (one per simulcast layer), which would be processed independently by the
+frame encryptor and assigned an unique CTR value for each.
 
-### SVC
+### Scalable Video Coding (SVC)
 
 In both temporal and spatial scalability, the SFU may choose to drop layers in
-order to match a certain bitrate or forward specific media sizes or frames per
+order to match a certain bitrate or to forward specific media sizes or frames per
 second. In order to support the SFU selectively removing layers, the sender MUST
 encapsulate each layer in a different SFrame ciphertext.
 
 ## Video Key Frames
 
-Forward Security and Post-Compromise Security require that the E2EE keys (base keys)
+Forward security and post-compromise security require that the E2EE keys (base keys)
 are updated any time a participant joins or leaves the call.
 
 The key exchange happens asynchronously and on a different path than the SFU signaling
 and media. So it may happen that when a new participant joins the call and the
 SFU side requests a key frame, the sender generates the E2EE frame
-with a key not known by the receiver, so it will be discarded. When the sender
+with a key that is not known by the receiver, so it will be discarded. When the sender
 updates his sending key with the new key, it will send it in a non-key frame, so
 the receiver will be able to decrypt it, but not decode it.
 
-The new Receiver will then re-request a key frame, but due to sender and SFU
+The new receiver will then re-request a key frame, but due to sender and SFU
 policies, that new key frame could take some time to be generated.
 
 If the sender sends a key frame after the new E2EE key is in use, the time
 required for the new participant to display the video is minimized.
 
 Note that this issue does not arise for media streams that do not have
 dependencies among frames, e.g., audio streams.  In these streams, each frame is
-independently decodeable, so there is never a need to process two frames
-together which might be on two sides of a key rotation.
+independently decodable, so there is never a need to process together two frames
```

Comment:
FWIW, I preferred the previous text, or perhaps "process two frames together that are on either side of a key rotation".

Reply:
Thanks, that's a better rewording.